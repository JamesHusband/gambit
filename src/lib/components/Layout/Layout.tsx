export const Layout = ({ children }: { children: React.ReactNode }) => (
  <div className="flex flex-col h-screen bg-[#242424]">{children}</div>
);
