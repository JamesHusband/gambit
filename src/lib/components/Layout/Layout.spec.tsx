import { render, screen } from "@testing-library/react";
import { Layout } from "./Layout";

describe("Layout", () => {
  it("should render with children", () => {
    const { container } = render(<Layout>Test</Layout>);
    expect(screen.getByText("Test")).toBeInTheDocument();
    expect(container.firstChild).toMatchSnapshot();
  });
});
