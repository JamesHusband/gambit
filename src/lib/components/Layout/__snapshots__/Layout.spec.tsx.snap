// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Layout should render with children 1`] = `
<div
  class="flex flex-col h-screen bg-[#242424]"
>
  Test
</div>
`;
